src/Equinox.Cosmos/Cosmos.fs

```diff
@@ -447,20 +440,62 @@ function sync(req, expIndex, expEtag) {
     }
 }"""
 
+module CancellationToken =
+    let useOrCreate = function Some ct -> async.Return ct | _ -> Async.CancellationToken
+
+type EquinoxCosmosClient (cosmosClient: CosmosClient, containerClient: CosmosContainer, databaseId: string, containerId: string) =
+    member val DatabaseId = databaseId with get
+    member val ContainerId = containerId with get
+    member val CosmosSdkClient = cosmosClient with get
+    member val ContainerSdkClient = containerClient with get
```

Comment:
this naming also seems cumbersome - if this is called CosmosStore / Context, maybe the Sdk bit can be implicit ?

Reply:
Sure, renamed and removed the "Sdk" part.